### ISSUE TITLE
Qubes R4.1rc4 installer tries to force EFI boot

### ISSUE THREAD

**Issue by GWeck** [QubesOS/qubes-issues#7235]
### Qubes OS release
Qubes R4.1-rc4

### Brief summary
When trying to install R4.1-rc4 with manual partitioning, the installer complains  that no EFI partition can be created with ext4 as file system, and the proposed partitioning scheme is GPT, not MBR.

### Steps to reproduce
During install of R4.1-rc4, select the mount points /boot (ext4), swap, and / (btrfs) with suitable sizes, intending to use legacy boot.

### Expected behavior
The selected partition scheme should be used. (This works in R4.1-rc3.)

### Actual behavior
The installer rejects the partitioning scheme, and installation cannot be started.

**Comment by marmarek** [comment]
Are you sure you started installation in legacy mode? The installer uses the running mode to determine whether EFI or legacy is used (and derives partition requirements from there).

**Comment by GWeck** [comment]
Indeed, I started the installer in EFI mode, unintentionally. On my system, the BIOS displays the installation media twice, in legacy and in EFI mode, and there I selected the wrong one. I now selected the USB stick in legacy mode, and the installation proceeded without error.

It might be helpful if the installer, at partition selection time, displayed the mode selected for installation, and better still, if it offered an option to change the mode.
